jupyterhub_singleuser_profiles/ui/src/ImageForm/imageUtils.ts

```diff
@@ -0,0 +1,13 @@
+import { ImageSoftwareType, ImageTagType } from '../utils/types';
+
+export const getNameVersionString = (software: ImageSoftwareType): string => {
+  const versionString = software?.version ? ` ${software?.version}` : '';
```

Comment:
comment/question... if version is always "truthy", or has a "V" in it, I'm good with this... any chance version will ever be `0`? 

probably don't need the `?` in ` ${software?.version}`, works regardless